yaml/lang/fr.yml

```diff
@@ -0,0 +1,430 @@
+    NoSkillsInCompendium: Le compendium de compétences configuré '{name}' ne contient aucune compétence. L'acteur a été créé sans listes de compétences initiales. Reconfigurez le recueil de compétences dans la configuration des options -> Genesys.
+    CannotDeleteCharacteristic: Impossible de supprimer l'entrée de journal XP pour la caractéristique '{characteristic}' - vous devez d'abord supprimer l'avancement au rang suppérieur !
+    CannotDeleteSkill: Impossible de supprimer l'entrée du journal d'XP pour la compétence '{name}' - vous devez d'abord supprimer l'entrée du passage au rang suivant.!"
+    CannotAffordTalent: "Impossible d'augmenter le talent '{name}' ({cost}xp)."
+    CannotAffordRankedTalent: "Impossible d'augmenter le talent '{name}' au rang {newRank} ({cost}xp)."
+    CannotDeleteNewTalent: Impossible de supprimer l'entrée du journal d'XP pour le nouveau talent '{name}' - Vous devez d'abord supprimer les augmentations de rang du talent !
+    CannotDeleteExistingTalent: "Impossible de supprimer l'entrée du journal de XP pour l'augmentation du talent '{name}' au rang {rank} - Vous devez d'abord supprimer les augmentations de rang supérieur !"
+    CannotPurchaseTalentTier: "Vous ne pouvez prendre des talents de la catégorie {tier} - vous devez avoir au moins {minimum} talents à la catégorie {lowerTier} d'abord"
+    NotEnoughStoryPoints: Vous ne pouvez pas dépenser de Point Narratif que vous ne possédez pas!
+    SelectOneTokenForAction: Vous devez selectionner un unique token pour réaliser cette action.
+    SelectNoneOrOneTokenForAction: Vous devez selectionner aucun token, ou juste un, pour faire cette action.
+    TokenIsNotCombatant: Le token selectionné n'est pas dans le combat actuel.
+    CannotClaimOppositeSlot: Vous ne pouvez par réclamer ce slot d'intiative avec {name} - il est du mauvais coté!
+    CannotSelectActor: Vous n'avez pas assez de droit pour choisir cet Acteur.
+    InvalidTokenTypeForAction: Le token selectionné est d'un type invalide pour réaliser cette action.
+
+  # Story Points
+  StoryPoints:
+    Player: Points Narratifs des Joueurs
+    GM: Points Narratifs du MJ
+    Reset: Réinitialise les Points Narratifs
+    Spend: Dépense de point
+    ChatMessage: Dépense un point Narratif!
+
+  # Localized strings used in TextEditor enrichers.
+  Enrichers:
+    Difficulty: 'Test de {skill} contre {difficulty} [{symbols}]'  # e.g. Average [♦♦] Coercion Check
+
+  # Combat Tracker
+  CombatTracker:
+    RevokeInitiativeClaim: Revoquer l'initiative
+
+  # Sheet Tab names
+  Tabs:
+    Skills: Compétence
+    Combat: Combat
+    Talents: Talents
+    Magic: Magie
+    Inventory: Inventaire
+    Effects: Effets
+    Journal: Journal
+    Description: Description
+    Data: Données
+    Details: Détails
+    Duration: Durée
+    Miscellaneous: Divers
+    Crew: Equipage
+    Attachments: attaches
```

Comment:
Missing proper capitalization?

Reply:
Yep, just to signal: i dont find in the game, and somme translation can be a  another word... 
I correct all capitalization problem you have signaled ;-), my appologize for that...
I also notice some other errors  that i have correct them...